docs/did-button-branding/index.md

```diff
@@ -0,0 +1,133 @@
+---
+layout: "doc"
+title: "Branding DID.app buttons."
+abstract: "DID.app provides several styles of icon that you can use to let people sign in."
+---
+
+<style>
+  .markdown-body img {
+    background: none;
+  }
+</style>
+
+You can use either our "D.I.D Logo" or "D Icon" to indicate to your users that they may sign in with DID.app.
+Both come in four styles; **default**, **light**, **dark-mono** and **light-mono**.
+
+### The D.I.D Logo
```

Comment:
```suggestion
### The DID.app Logo
```

Reply:
Yea, I think the DID logo and the D icon can be distinct things.  The D icon isn't our logo but it does fit style wise with our logo if you see what I mean.

I just don't like the dots between the letters.